util/binfmt_misc/check.go

```diff
@@ -35,5 +36,13 @@ func check(bin string) error {
 	}
 	f.Close()
 
-	return exec.Command(pp).Run()
+	err = exec.Command(pp).Run()
+	if err == nil {
+		cmd := exec.Command("/check")
```

Comment:
what's the benefit of running it twice and not only the chroot version?

Reply:
Nice catch! no benefit at all, new RP is one the way.